_includes/homepage-newsletter-band.html

```diff
@@ -0,0 +1,9 @@
+<div class="full-width-bg component homepage-content-band">
```

Comment:
are you missing some files ? i dont see this new file referenced anywhere so it does not show up anywhere.

Reply:
added the band to the index page... I must have unchecked in my original submission.